src/Experiment/src/AssemblyBuilder.cs

```diff
@@ -10,22 +9,20 @@
 namespace System.Reflection.Emit.Experimental
 {
 
-    public class AssemblyBuilder: System.Reflection.Assembly
+    public class AssemblyBuilder : System.Reflection.Assembly
     {
-        private bool _previouslySaved=false;
-        private AssemblyName _assemblyName;
-        private MetadataBuilder _metadata;
+        private bool _previouslySaved = false;
+        private AssemblyName? _assemblyName;
```

Comment:
Why do we need to add nullable annotation here when the only constructor doesn't take a nullable `AssemblyName`?